modules/ivc_champva/app/controllers/ivc_champva/v1/uploads_controller.rb

```diff
@@ -23,11 +23,12 @@ def submit
 
           status, error_message = FileUploader.new(form_id, metadata, file_paths, attachment_ids, true).handle_uploads
 
-          render json: build_json(Array(status), error_message)
+          render json: build_json(Array(status), error_message), status:
```

Comment:
Previously the status was contained in the JSON made by `build_json`, but that didn't actually set the status code on the wrapper HTTP response object - this explicitly sets that status to match the inner status so errors or other status codes should come through